docs/reference/mapping/types/numeric.asciidoc

```diff
@@ -78,6 +78,11 @@ to help make a decision.
 |`half_float`|+2^-24^+ |+65504+ |+11+ / +3.31+
 |=======================================================================
 
+[[number-array]]
+==== Array handling for numeric fields
```

Comment:
I'm not sure it makes sense to add references about array handling to just these two pages (`numeric` and `text`), because it could imply that other field types cannot be arrays. It is also fairly arbitrary to include notes on these two pages -- it is common to use arrays for many other types like `keyword`, `date`, or `ip`. I'm curious as to your opinions here.

Reply:
I added notes to `text` and `numeric` based on the description of this issue: 43892.

However, if arrays are commonly used with many datatypes, it might make most sense to not include these notes at all. I'm not very familiar with the usage trend so I'll defer to you.

In the meantime, I removed these notes with e8f6d42.

Reply:
Thanks, I think the change looks good with these notes omitted.